docs/source/en/_toctree.yml

```diff
@@ -13,6 +11,8 @@
     title: Overview
   - local: using-diffusers/write_own_pipeline
     title: Understanding models and schedulers
+  - local: stable_diffusion
```

Comment:
Shouldn't we give a bit less generic name to this doc since we're discussing optimization specifically?

Reply:
Yeah and this way we don't break any links to this doc!

Reply:
I'd actually like to keep it a main doc page for now